pac4j-couch/src/main/java/org/pac4j/couch/profile/CouchProfile.java

```diff
@@ -0,0 +1,15 @@
+package org.pac4j.couch.profile;
+
+import org.pac4j.core.profile.CommonProfile;
+
+/**
+ * <p>The user profile returned from a MongoDB.</p>
```

Comment:
The Javadoc refers to MongoDB, I guess there is a copy/paste which must be updated here.